HelpSource/Classes/Server.schelp

```diff
@@ -774,3 +774,6 @@ argument:: numChannels
 Number of channels to send to BelaScope, starting from the index supplied. Defaults to 2, or to link::Classes/ServerOptions#-numOutputBusChannels:: if index is 0.
 
 returns:: A link::Classes/Synth::, linking this Server's desired bus channels to BelaScope's bus.
+
+subsection:: Server issues
+Some insights about common Server issues can be found on the FAQ link::Guides/UserFAQ#Server Issues#Server Issues::
```

Comment:
I think this is hard to notice if it's towards the end of the document. I think this could to at the end of the *Descirption*, maybe even without the `subsection...`, just the sentence "Some insights..."; additionally, that link could also be added to the "See also" section at the top.